applications/vanilla/controllers/api/DiscussionsApiController.php

```diff
@@ -222,6 +227,11 @@ public function index(array $query) {
 
         $in = $this->schema([
             'categoryID:i?' => 'Filter by a category.',
+            'pinned:b?' => 'Whether or not to include pinned discussions. If true, only return pinned discussions.',
+            'pinOrder:s?' => [
+                'enum' => ['first', 'mixed'],
+                'description' => 'If including pinned posts, in what order should they be integrated?'
```

Comment:
The description needs an update. It should state that this filter will only work if `pinned` is not used.
There should also probably be a default value set to 'mixed' since that's how discussions are returned if no filter is used.